package.json

```diff
@@ -36,7 +36,10 @@
     "lint:getting-started:fix": "eslint ./getting-started/**/*.{js,ts} --fix",
     "lint:markdown": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md",
     "lint:markdown:fix": "./node_modules/.bin/markdownlint $(git ls-files '*.md') -i ./CHANGELOG.md --fix",
-    "reset": "lerna clean -y && rm -rf node_modules && npm i && npm run compile && npm run lint:fix"
+    "reset": "lerna clean -y && rm -rf node_modules && npm i && npm run compile && npm run lint:fix",
+    "update-ts-references": "update-ts-references --configName tsconfig.esm.json && update-ts-references",
+    "use:stable": "ln -sf lerna-stable.json lerna.json",
```

Comment:
i think we should note somewhere that the tooling is made for unix based env since i believe ln isn't available on newer windows version ?

Reply:
I could use cp i suppose if you think thats better

Reply:
I would also prefer a tooling working on windows. for deleting trees rimraf can be used and I guess there are similar tools for other operations.

Reply:
Does cp work on windows (sorry I'm not familiar with recent versions)?

Reply:
not out of the box - it's named copy but has different options.

Reply:
maybe https://www.npmjs.com/package/shelljs would be helpful